test/epgsql_replication_SUITE.erl

```diff
@@ -1,110 +1,174 @@
 -module(epgsql_replication_SUITE).
+
 -include_lib("eunit/include/eunit.hrl").
 -include_lib("common_test/include/ct.hrl").
 -include("epgsql.hrl").
 
--export([
-    init_per_suite/1,
-    all/0,
-    end_per_suite/1,
+-export([ all/0
+        , init_per_suite/1
+        , end_per_suite/1
 
-    connect_in_repl_mode/1,
-    create_drop_replication_slot/1,
-    replication_sync/1,
-    replication_async/1,
+        , connect_in_repl_mode/1
+        , create_drop_replication_slot/1
+        , replication_sync/1
+        , replication_async/1
+        , replication_async_active_n_socket/1
+        , replication_sync_active_n_socket/1
 
-    %% Callbacks
-    handle_x_log_data/4
-]).
+          %% Callbacks
+        , handle_x_log_data/4
+        , handle_socket_passive/1
+        ]).
 
 init_per_suite(Config) ->
-    [{module, epgsql}|Config].
+  [{module, epgsql} | Config].
 
 end_per_suite(_Config) ->
-    ok.
+  ok.
 
 all() ->
-    [
-     connect_in_repl_mode,
-     create_drop_replication_slot,
-     replication_async,
-     replication_sync
-    ].
+  [ connect_in_repl_mode
+  , create_drop_replication_slot
+  , replication_async
+  , replication_sync
+  , replication_async_active_n_socket
+  , replication_sync_active_n_socket
+  ].
```

Comment:
please let's not re-format everything!

Reply:
I didn't review this file yet because it's not clear what exactly have changed...

Reply:
It was 80-90% changed. Don't see a reason to keep the rest unchanged ( ;